templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -423,6 +423,14 @@ server {
 {%             for port in nginx_tpl_listen              %}
         listen {{ port|string }};
 {%             endfor                                    %}
+{%             if nginx_tpl_ssl | bool                   %}
+{%                 for port in nginx_tpl_listen_ssl      %}
+        listen {{ port|string }};
```

Comment:
This line [according to the documentation](https://nginx.org/en/docs/http/configuring_https_servers.html#single_http_https_server) should be:
```
listen {{ port|string }} ssl;
```
Can you check if this works as expected?